svvp.json

```diff
@@ -3,8 +3,7 @@
   "drivers": [
     "NetKVM",
     "vioscsi",
-    "fwcfg64",
-    "tpm"
+    "fwcfg64"
```

Comment:
tpm is required for SVVP, don't remove it from this list, we should fix the bug with tpm duplication instead